platform/backend/repository/memory/src/main/kotlin/io/hamal/repository/memory/record/Trigger.kt

```diff
@@ -201,14 +205,39 @@ class MemoryTriggerRepository : MemoryRecordRepository<TriggerId, TriggerRecord,
                         name = cmd.name,
                         inputs = cmd.inputs,
                         cron = cmd.cron,
-                        correlationId = cmd.correlationId
+                        correlationId = cmd.correlationId,
+                        status = cmd.status
                     )
                 )
                 (currentVersion(triggerId) as CronTrigger).also(CurrentTriggerProjection::apply)
             }
         }
     }
 
+    override fun set(triggerId: TriggerId, cmd: SetTriggerCmd): Trigger {
+        return lock.withLock {
+            if (commandAlreadyApplied(cmd.id, triggerId)) {
+                versionOf(triggerId, cmd.id)
+            } else {
+                val rec: TriggerRecord = if (cmd.status == TriggerStatus.Active) {
+                    ActiveTriggerRecord(
```

Comment:
```suggestion
                    TriggerSetActiveRecord(
```

vice versa for inactive